docs/quickstart.rst

```diff
@@ -18,8 +18,7 @@ Installation
 2. Add ``zgw_consumers`` to the ``INSTALLED_APPS`` setting.
 3. Run ``python src/manage.py migrate`` to create the necessary database tables
 4. Configure `django-privates <https://django-privates.readthedocs.io/en/latest/quickstart.html>`_
```

Comment:
I'd point to django-simple-certmanager instead - zgw-consumers itself doesn't need django-privates as dependency anymore I think.

Reply:
I agree